### PR TITLE
Updated to Android Gradle Plugin 4.1.1 and Gradle 6.7.1

### PR DESCRIPTION
**Pull request by jkasten2**
**Recommend reviewing commit-by-commit**

* Simple maintenance version bump to latest Gradle and AGP.
* Similar actions to PR #139

### CODE REVIEW

File: src/test/groovy/com/onesignal/androidsdk/MainTest.groovy [hunk 1]
```diff
@@ -1295,11 +1295,11 @@ class MainTest extends Specification {
         when:
         def results = runGradleProject([
             compileLines: """
-                compile 'com.android.support:appcompat-v7:26.0.0'
-                compile 'com.google.firebase:firebase-messaging:15.0.2'
-                compile 'com.google.firebase:firebase-core:16.0.0'
+    implementation 'com.google.android.gms:play-services-base:15.0.1'
```
Comment (Jeasmine) [inline src/test/groovy/com/onesignal/androidsdk/MainTest.groovy:1298]: is this indentation ok?

Comment (jkasten2) [reply src/test/groovy/com/onesignal/androidsdk/MainTest.groovy]: ops, didn't mean to commit this at all actually. Was just using this to test something.
Created fixup commit de91e2e59f788b4e39857a4f062759178c74849a to undo this change.

Comment (jkasten2) [reply src/test/groovy/com/onesignal/androidsdk/MainTest.groovy]: Ops messed up one line in that fixup commit so just force push that fixup as fe5ae0792c7826394a42e77a2c4af76a6b15d512.
Sorry done messing with it, should be good now

**Review by Jeasmine — APPROVED** [review]

**Comment by jkasten2** [comment]
Ran `git rebase master -i --autosquash` to clean up fixup now that this PR is approved

**Pull request merged**